components/Memory.js

```diff
@@ -40,6 +32,19 @@ export const Card = styled.section`
         backface-visibility: hidden;
         height: 100%;
     }
+    
+    .card-0 {
+        transform: rotateY(0deg);
+    }
+    
+    &:active {
```

Comment:
```suggestion
    &.active {
```

Reply:
thanks for the help, but the solution was to change the pseudo class of after :D